src/swsssdk/scripts/sonic-db-cli

```diff
@@ -1,30 +1,50 @@
 #!/usr/bin/python
 import sys
 import swsssdk
+import redis
+import syslog
+import argparse
+from multiprocessing import Pool
 
-argc = len(sys.argv)
-if argc == 2 and sys.argv[1] == '-h':
-    print("""
-Example 1: sonic-db-cli CONFIG_DB keys *
-Example 2: sonic-db-cli APPL_DB HGETALL VLAN_TABLE:Vlan10
-Example 3: sonic-db-cli APPL_DB HGET VLAN_TABLE:Vlan10 mtu
-Example 4: sonic-db-cli APPL_DB EVAL "return {KEYS[1],KEYS[2],ARGV[1],ARGV[2]}" 2 k1 k2 v1 v2
-""")
-elif argc < 3:
-    msg = "'Usage: sonic-db-cli <db_name> <cmd> [arg [arg ...]]'. See 'sonic-db-cli -h' for detail examples."
-    print >> sys.stderr, msg
-else:
-    dbname = sys.argv[1]
+def ping_single_instance(inst_info):
+    inst_hostname = inst_info['hostname']
+    inst_port = inst_info['port']
+    r = redis.Redis(host=inst_hostname, port=inst_port)
+    rsp = False
+    msg = 'Could not connect to Redis at {}:{}: Connection refused'.format(inst_hostname, inst_port)
+    try:
+        rsp = r.ping()
+    except redis.exceptions.ConnectionError as e:
+        syslog.syslog(syslog.LOG_ERR, 'ping redis inst failed at host {} port {}'.format(inst_hostname, inst_port))
```

Comment:
>syslog.syslog(syslog.LOG_ERR, 'ping redis inst failed at host {} port {}'.format(inst_hostname, inst_port)) [](start = 8, length = 107)

Since the message is already on stdout, no gain to write syslog. Also to mimic redis-cli. Could you remove this line?

Reply:
removed